rollouts/PROJECT

```diff
@@ -22,4 +22,13 @@ resources:
   kind: RemoteRootSync
   path: github.com/GoogleContainerTools/kpt/rollouts/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
```

Comment:
I'm questing if we actually need a controller for the ProgressiveRolloutStrategy.

Reply:
As discussed offline, we plan to use it for the following:
- Aggregating what rollouts are using this strategy
- Finalizer to prevent deletion of a strategy in case it is being used by any rollouts.